### PR TITLE
Add tooltips to Cluster, Logarithmic, and Linear buttons to complete user story #4

### LINKED ISSUE #4: Add tooltips to the buttons that are missing them i.e. the log, cluster, linear, etc. graph buttons. Justification: Tooltips provide additional context or description about the purpose or function of a button. This is particularly useful for buttons with non-obvious functions or icons, helping users understand what action will be performed when the button is clicked. Tooltips can improve accessibility by providing additional information for users who may have difficulty interpreting visual elements alone. For example, users with visual impairments who rely on screen readers can benefit from tooltips that describe button functionality.

### LINKED ISSUE #5: Tooltip: Write JavaScript code to handle the tooltip behaviors

### LINKED ISSUE #6: Tooltip: Style your tooltips using CSS to control their appearance

### LINKED ISSUE #7: Tooltip: Create a tooltip HTML class for log, cluster, and linear buttons to improve accessibility

### LINKED ISSUE #8: Tooltip: Decide the content we want to display with the tooltips

### PR DESCRIPTION
**Pull request by asturnley**
# Pull request to resolve user story #4
### What?
Tooltips are descriptors for buttons that can help users understand what each button does. These were added to buttons that did not have existing tooltips in the JPlag.
___

### Why?
Tooltips can help inform users about the usage of buttons and labels. They can also improve accessibility within the application as now accessibility readers are able to inform users of the buttons and their uses. The linear, logarithmic, and clusters buttons did not have existing tooltips like other buttons so it was important to add tooltips to those as well.
___

### How?
To locate the buttons, the Inspect tool was used to find the names of the buttons and labels. The Linear and Logarithmic labels were found in the _report-viewer/src/views/OverviewView.vue_ file and the Clusters button was found in the _report-viewer/src/components/ComparisonTableFilter.vue_ file. In each instance the tooltip information was passed to the corresponding tooltip template file to create the tooltip. The information passed to the file was the `displayValue` for the button or label (i.e., linear, logarithmic, and clusters) and the tooltip content. For the cluster button, the information had to be added to the specifically if the cluster sorting was enabled.
___

### Where?
**Files modified:**

- report-viewer/src/components/ComparisonTableFilter.vue
- report-viewer/src/views/OverviewView.vue

**This pull request resolves #4 user story by:** 
- Resolve issue #5 : Implementing threshold list functionality/filtering by threshold value
- Resolve issue #8 : Write JavaScript code to handle tooltip behaviors
___

### TL;DR:
Ultimately, the tooltips were called into the tooltip template files provided for the Cluster, Linear, and Logarithmic buttons. These template files were included in the original artifact thus providing the HTML classes and CSS design for the tooltips, which resolves issue #6 and #7.

### CODE REVIEW

**Review by Jerry4753 — APPROVED** [review]
I see that you figured out how to add tooltip to OptionsSelector after a few attempts. Good work

**Comment by JavaNoTea** [comment]
Resolves #4
